sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/DescribeTableSuite.scala

```diff
@@ -213,4 +213,29 @@ class DescribeTableSuite extends command.DescribeTableSuiteBase
       assert("""\d+\s+bytes,\s+4\s+rows""".r.matches(stats))
     }
   }
+
+  Seq(true, false).foreach { hasCollations =>
```

Comment:
what's the difference between this one and the one in `v1/DescribeTableSuite`? Can we move it up to the parent suite?

Reply:
Moved the tests to DescribeTableSuiteBase suite, to avoid code duplication, whereas these test cases are now run as part of both v1 and v2 suites for complete coverage.